account_bank_statement_line_reconciliation/README.rst

```diff
@@ -0,0 +1,66 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```

Comment:
Readme should be like this now [https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst](https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst)

Reply:
Did new readme guide apply for an older version?

Reply:
I'm not sure but is not written "only for version >= 11"

Reply:
@Tonow-c2c What's wrong with this readme?

Reply:
Now the readme is on multi parts like you cane see 
[https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst](https://github.com/OCA/maintainer-tools/blob/master/template/module/README.rst)
[https://github.com/OCA/maintainer-tools/tree/master/template/module/readme](https://github.com/OCA/maintainer-tools/tree/master/template/module/readme)